src/components/App.js

```diff
@@ -15,6 +18,7 @@ import './App.scss';
 
 type Props = {
   isTeamSidebarOpen: boolean,
+  loggedIn: boolean,
```

Comment:
Why do you need loggedIn? You do not seem to use it :D
The custom route you created should do the trick for you.